examples/remove-objects.js

```diff
@@ -18,27 +18,27 @@
 // replace them with original values.
 
 
-var Minio = require('minio');
+var Minio = require('minio')
 
 var s3Client = new Minio.Client({
   endPoint: 's3.amazonaws.com',
   port: 9000,
   useSSL: false,
   accessKey: 'YOUR-ACCESSKEYID',
   secretKey: 'YOUR-SECRETACCESSKEY'
-});
+})
 
 var objectsList = []
 
 // List all object paths in bucket my-bucketname.
 var objectsStream = s3Client.listObjects('my-bucketname', '', true)
```

Comment:
Assuming the other 2 suggested changes/comments are accepted, this example script can be simplified to work for both versioned and non-versioned scenarios:
```
var objectsList = []
var bucket = '<bucket-name>'
var prefix = '<prefix-name>'
var recursive = <true | false>
var includeVersion = <true | false>

// List all object paths in bucket
var objectsStream = s3Client.listObjects(bucket, prefix, recursive, {IncludeVersion: includeVersion})

objectsStream.on('data', function(obj) {
  if (includeVersion) {
    objectsList.push(obj)
  } else {
    objectsList.push(obj.name)
  }
})

objectsStream.on('error', function(e) {
  return console.log(e)
})

objectsStream.on('end', function() {
  s3Client.removeObjects(bucket, objectsList, function(e) {
    if (e) {
      return console.log(e)
    }
    console.log("Success")
  })
})
```

Reply:
Thank you for the suggestion. I will refactor.